firebase-firestore/src/androidTest/java/com/google/firebase/firestore/WriteBatchTest.java

```diff
@@ -306,4 +306,31 @@ public void testRunBatch() {
     assertTrue(snapshot.exists());
     assertEquals(map("foo", "bar", "baz", 42L), snapshot.getData());
   }
+
+  @Test
+  public void testCanHandlePendingMutationsWithIrrelevantDocuments() {
```

Comment:
Can you use a different term for `irrelevant` here? 

`testCanHandleBatchThatCausesPartialViewRecalculation()` (maybe)

Reply:
Renamed.